microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md

```diff
@@ -0,0 +1,180 @@
+## Sources tool now notifies you when sourcemaps can't be loaded
+
+<!-- 4 -->
+
+<!-- Title: Get notified when DevTools cannot load your sourcemaps correctly -->
+<!-- Subtitle: The Sources tool now provides several places in the UI when DevTools can't fetch or parse your sourcemaps. -->
+
+In Microsoft Edge version 96, the **Sources** tool now provides several indications in the UI when DevTools cannot load your sourcemaps.  In the **Page** tab in the **Navigator** pane in the **Sources** tool, files that DevTools can't load sourcemaps for have a warning<!--todo: capture/show--> icon in place of the file<!--todo: capture/show--> icon.  
+
+Selecting a file with the warning icon opens the file in the Sources tool with an infobar<!--todo: capture/show--> indicating that DevTools either couldn't fetch the sourcemap correctly from the server or couldn't parse the sourcemap correctly.
+
+From the infobar, you can learn more about the issue by selecting the **Open in Issues** button.  The **Issues** tool then opens in the **Drawer** and provides information about how to resolve the issue and get your sourcemap loaded correctly in DevTools.
+
+* To prevent the infobar about sourcemaps from cluttering the **Sources** tool, select the **Don't show again** button.
+* To prevent issues related to sourcemaps from cluttering the **Issues** tool, clear the **Include third-party issues** checkbox in the **Issues** tool.
+
+:::image type="content" source="../../media/2021/11/source-map-not-found.png" alt-text="The Sources tool now notifies you in the Issues tab when sourcemaps can't be loaded.":::
+
+<!-- video: see work item -->
+
+To learn more about how DevTools fetches and parses sourcemaps, see [Map preprocessed code to source code](../../../javascript/source-maps.md).
+
+
+<!-- ====================================================================== -->
+## Opening source files in Visual Studio Code now integrates better with the Sources tool
+
+<!-- 5 -->
+
+<!-- Title: Open source files directly in Visual Studio Code from DevTools -->
+<!-- Subtitle: The "Open source files in Visual Studio Code" experiment now works more intuitively with the Sources tool. -->
+
+In previous versions of Microsoft Edge, turning on the **Open source files in Visual Studio Code** experiment had unexpected behavior when you had already been using the **Sources** tool in DevTools.  Setting breakpoints would direct you to Visual Studio Code or to documentation to configure the experiment correctly.
+
+In Microsoft Edge version 96, the **Open source files in Visual Studio Code** experiment now integrates better with the Sources tool.
+
+*  If the Sources tool is open and you then turn on the **Open source files in Visual Studio Code** experiment, setting breakpoints or opening files now opens the Sources tool, rather than directing you to Visual Studio Code or to documentation to configure the experiment correctly.
```

Comment:
```suggestion
*  If the **Sources&& tool is open and you then turn on the **Open source files in Visual Studio Code** experiment, setting breakpoints or opening files now opens the **Sources** tool, rather than directing you to Visual Studio Code or to documentation to configure the experiment correctly.
```